src/core/Akka.Remote/Endpoint.cs

```diff
@@ -1140,7 +1074,7 @@ private void AdjustAdaptiveBackup()
             }
             else if (_writeCount >= _maxWriteCount * 0.6)
             {
-                _adaptiveBackoffNanos = Math.Max(Convert.ToInt64(_adaptiveBackoffNanos * 0.9), MaxAdaptiveBackoffNanos);
+                _adaptiveBackoffNanos = Math.Max(Convert.ToInt64(_adaptiveBackoffNanos * 0.9), MinAdaptiveBackoffNanos);
```

Comment:
Fixes a long-standing bug in this code where I used the wrong constant during adaptive backoff calculation. Found it because ReSharper indicated that the`MinAdaptiveBackoffNanos` value was not being used.